autoscheduler/frontend/src/components/NavBar/NavBar.css

```diff
@@ -1,4 +1,9 @@
-.name-and-button {
+.logout-container {
     display: flex;
     align-items: center;
+    min-width: 0;
```

Comment:
Why is this necessary?

Reply:
It's here because the text for login/logout used to shrink just like the last updated text, but I forgot to remove this line when I changed it